PRODUCTION.md

```diff
@@ -0,0 +1,82 @@
+# Using S3 for continuous integration testing with Docker
+
+When you start the Scality S3server image, you can adjust the configuration of
+the Scality S3server instance by passing one or more environment variables
+on the docker run command line.
+
+## Environment Variables
+
+### HOST_NAME
+
+This variable specifies an host name:
```

Comment:
Yes @rahulreddy for hosted style but it is also true for path-style since if the host name is not listed it will return `throw new Error('bad request: hostname ${host} is not in valid endpoints');`. I just wanted to be sure.